utils/sub8_diagnostics/CMakeLists.txt

```diff
@@ -1,18 +1,67 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(sub8_diagnostics)
 
-find_package(catkin REQUIRED COMPONENTS
+find_package(catkin
+ REQUIRED COMPONENTS
+  roscpp
   rospy
   mil_tools
+  message_runtime
+  sub8_msgs
+  mil_msgs
+  actionlib
+  message_generation
+  geometry_msgs
+  sensor_msgs
+  interactive_markers
+  std_msgs
+  actionlib_msgs
+  mil_tools
+)
```

Comment:
Only some of these dependencies are actually needed (check the includes in your code). While adding more doesn't change program behavior, it does complicate the build process and make it more confusing to read